site/content/blog/2015/03/15/hockey-stick-growth/index.txt

```diff
@@ -0,0 +1,17 @@
+---
+title: Hockey Stick Growth
+created_at: 2015-03-15 20:02:27.000000 -05:00
+tags:
+- 2015
+blog_post: true
+filter:
+- erb
+- markdown
+---
+
+The explosion of devopsdays events in recent years is exciting for everyone in the community. Some of us have been in this space since before the first event in Ghent, while others are joining with the newest events coming online in 2015. In the last year we've added new global core organizers, seen the originator of devopsdays (Patrick Debois) move into an advisory role, and been thrilled by rapid expansion in the numbers and locations of events. 
+
+If you'd like to host a devopsdays where you live, it's easy to get started. Just talk to the members of your local devops community and get a few people together from different organizations so you have an assortment of perspectives. Then check out our [organizing guide](http://www.devopsdays.org/pages/organizing/) and drop us a line letting us know you'd like to join in!
+
+
+Bridget Kromhout
```

Comment:
I like the idea of having more names here, so if you'd like to add anything to this blog post, please comment and we can have multiple authors. (Being specific is also nice, though, so I want names.)